cmd/mysql/binlog_push.go

```diff
@@ -28,4 +30,5 @@ var binlogPushCmd = &cobra.Command{
 
 func init() {
 	cmd.AddCommand(binlogPushCmd)
+	binlogPushCmd.Flags().StringVar(&untilBinlog, "until", "", "binlog file name to stop at. Current active by default")
```

Comment:
It is hard to read `Current active by default` for me. I can suggest following description: `When omitted WAL-G uploads binlogs up to current active binlog.`